src/network/connection.cpp

```diff
@@ -825,8 +825,9 @@ void Peer::DecUseCount()
 	delete this;
 }
 
-void Peer::RTTStatistics(float rtt, const std::string &profiler_id,
-		unsigned int num_samples) {
+void Peer::RTTStatistics(float rtt, const std::string &profiler_id)
+{
+	static const float avg_factor = 0.1 / MAX_RELIABLE_WINDOW_SIZE;
```

Comment:
I didn't use a float constant because l think the number is calculated during compile time, so a double division followed by a float conversion would be more precise.